groups/script.js

```diff
@@ -76,46 +90,85 @@ const memberAddRoleBody = {
  *
  * FOR RENDERING GROUP ROLES IN 'MANAGE ROLES' TAB
  */
-const groupsData = await getDiscordGroups();
+let groupsData = await getDiscordGroups();
```

Comment:
Why using `let` ? Don't see the variable changing.

Reply:
Done.